src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -692,12 +673,12 @@ private void ParseHeaderNameValue(Span<byte> line, HttpResponseMessage response)
             }
 
             // Skip whitespace after colon
-            while (pos < line.Length && (line[pos] == (byte)' ' || line[pos] == '\t'))
+            while (pos < line.Length && (line[pos] == (byte)' ' || line[pos] == (byte)'\t'))
```

Comment:
Was the `(byte)` actually needed, or just for consistency?

Reply:
Consistency.  That said, we're not consistent in other places here.